pbiapi/pbiapi.py

```diff
@@ -352,9 +352,66 @@ def import_file_into_workspace(
             else:
                 logging.info("Import in progress...")
 
+    @check_token
+    def update_parameters_in_dataset(self, workspace_name: str, dataset_name: str, parameters: list):
+        workspace_id = self.find_entity_id_by_name(self.workspaces, workspace_name, "workspace", raise_if_missing=True)
+
+        datasets = self.get_datasets_in_workspace(workspace_name)
+        dataset_id = self.find_entity_id_by_name(datasets, dataset_name, "dataset", raise_if_missing=True)
```

Comment:
This operation is done multiple times and I suggest that it is moved into its own method.